app/controllers/index.js

```diff
@@ -1,23 +1,23 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-    async = require('async'),
-    _ = require('underscore');
 
 /**
  * Redirect users to /#!/app (forcing Angular to reload the page)
+ * @param {obj} req express request object
+ * @param {obg} res express response object
+ * @returns {redirect} redirect user
  */
-exports.play = function(req, res) {
+exports.play = (req, res) => {
```

Comment:
Since you have babel now, you can change this to es6 `export`. Change all instances in this PR.